server.js

```diff
@@ -1,118 +1,101 @@
 const express = require('express');
 const app = express();
-const bodyParser = require('body-parser');
 const mongoose = require('mongoose');
 const Pet = require('./models/pet.js');
+const bodyParser = require('body-parser');
 
-mongoose.connect('mongodb://localhost:27017/updog');
+const dbURL = process.env.MONGODB_URI || 'mongodb://localhost/updog'
+
+mongoose.connect(dbURL);
```

Comment:
This is provided on Heroku for you yes. It will just be there when you include certain add-on. Notes to follow